src/hoc/RealTime.tsx

```diff
@@ -48,6 +51,15 @@ export const RealTime = ({ children, task }: { children: ReactNode; task?: TaskR
           }
           //if the task is updated
         } else {
+          if (oldTask && oldTask.body && updatedTask.body) {
+            const oldImgSrcs = extractImgSrcs(oldTask.body)
+            const newImgSrcs = extractImgSrcs(updatedTask.body)
+
+            if (oldImgSrcs.length > 0 && newImgSrcs.length > 0) {
+              updatedTask.body = replaceImgSrcs(updatedTask.body, newImgSrcs, oldImgSrcs)
```

Comment:
Need to extract new image Srcs and replace it with old ones, because since we are creating a new url of images on each task details navigation, a second user navigating the task details will generate a new src and replace it in the database which causes the previous user to load the src again(because its new)

Added a comment in the code block. thanks.